src/core/DOMEval.js

```diff
@@ -3,12 +3,19 @@ define( [
 ], function( document ) {
 	"use strict";
 
-	function DOMEval( code, doc ) {
+	function DOMEval( code, doc, node ) {
 		doc = doc || document;
 
-		var script = doc.createElement( "script" );
-
+		var i,
+			script = doc.createElement( "script" );
 		script.text = code;
+		if ( node ) {
+			for ( i in { type: true, src: true } ) {
```

Comment:
This object should be a `preservedScriptAttributes` variable, not a literal buried in the function body.